src/lib/iframe-tools/identity-manager.js

```diff
@@ -59,27 +64,83 @@ export class IdentityManager extends IframeManager {
     this.completionListener(identity, null);
   }
 
+  fetchCountry = () => {
+    return new Promise((resolve) => {
+      this.addMessageListener(({ data: { type, payload } }) => {
```

Comment:
Should we unregister this message listener after getting the payload, or is fetchCountry only expected to run once and it wouldn't make much of a difference? Same question for fetchUserInfo

Reply:
@luisingho This is a good point, I would be happy to do it. Unfortunately, current implementation of message manager doesn't allow it.
I can't include it in current pull request because it is way outside scope of this task.